src/slides/class1.jsx

```diff
@@ -1258,5 +1258,173 @@ export default () => (
+          Si no necesitamos detalles sobre el error podemos <b>omitir el parámetro error en el bloque catch. </b>
+          Además, pueden existir casos donde deseemos lanzar un error de manera intencional por comportamiento imprevisto
+          en el código. Para esto utilizamos la palabra clave <b>throw</b>.
+        </Text>
+        <CodePane language="js">
+            {`
+              let json = '{ "age": 30 }'; // incomplete data
+
+              try {
+                let user = JSON.parse(json); // <-- no errors
+                if (!user.name) {
+                  throw new SyntaxError("Incomplete data: no name");
+                }
+                alert( user.name );
+              } catch (err) {
+                alert( "JSON Error: " + err.message ); // JSON Error: Incomplete data: no name
+              }
+            `}
+        </CodePane>
+      </div>
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          Manejo de errores (cont.)
+        </Heading>
+        <Text fontSize={30}>
+          Para lanzar errores propios JavaScript nos provee varios constructores para errores comúnes: Error, SyntaxError, ReferenceError, TypeError y otros.
+          Su sintáxis es la siguiente:
+        </Text>
+        <CodePane language="js">
+            {`
+              let error = new Error(message);
+              // or
+              let error = new SyntaxError(message);
+              let error = new ReferenceError(message);
+              // ...
+            `}
+        </CodePane>
+        <Text fontSize={30}>
+          En estos casos, la propiedad <b>name</b> tendrá como valor el nombre del constructor (ej. SyntaxError) y la propiedad <b>message</b> tendrá como valor lo que hayamos pasado como parámetro al constructor.
+        </Text>
+      </div>
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          Manejo de errores (cont.)
+        </Heading>
+        <Text fontSize={30}>
+          Los bloques <b>try..catch</b> pueden contar con una cláusula extra (opcional) llamada <b>finally</b>.
+          Si está presente, el código contenido dentro de ese bloque se ejecutará siempre:
+          <UnorderedList fontSize={30}>
+            <ListItem>Después del <b>try</b> si no hubo errores.</ListItem>
+            <ListItem>Después del <b>catch</b> si hubo errores.</ListItem>
+          </UnorderedList>
+        </Text>
+        <CodePane language="js">
+            {`
+              try {
+                ... try to execute the code ...
+             } catch (err) {
+                ... handle errors ...
+             } finally {
+                ... execute always ...
+             }
+            `}
+        </CodePane>
+      </div>
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          Creando errores custom
```

Comment:
```suggestion
          Creando nuestro propio error
```
We avoid avoid spanglish here 😆